### PR TITLE
Add ESLint

### PR DESCRIPTION
**Pull request by rafenden**

### CODE REVIEW

File: src/activity-feed/activity-feed-cards/card/CardUtils.jsx [hunk 1]
```diff
@@ -29,9 +33,9 @@ const createJobTitleMarkup = ({id, url, name, jobTitle}) => {
 }
 
 const getPeople = (activity, personSubType) => {
-  return map(filter(activity['object']['attributedTo'], ({type}) => {
+  return map(filter(activity.object.attributedTo, ({ type }) => {
```
Comment (paulgain) [inline src/activity-feed/activity-feed-cards/card/CardUtils.jsx]: Consider using `  const { attributedTo } = activity.object` for consistency.

File: src/activity-feed/activity-feed-cards/InteractionUtils.js [hunk 2]
```diff
@@ -41,7 +45,9 @@ const isServiceDelivery = (activity) => {
 export default class CardUtils {
   static transform(activity) {
     const status = getStatus(activity)
-    const badge = isServiceDelivery(activity) ? BADGE_LABELS.COMPLETED_SERVICE_DELIVERY : BADGE_LABELS[status.toUpperCase()]
+    const badge = isServiceDelivery(activity)
+      ? BADGE_LABELS.COMPLETED_SERVICE_DELIVERY
+      : BADGE_LABELS[status.toUpperCase()]
```
Comment (paulgain) [inline src/activity-feed/activity-feed-cards/InteractionUtils.js]: If `status` is `null` this line will throw a `TypeError` when calling `toUpperCase()`.

File: src/activity-feed/activity-feed-cards/InteractionUtils.js [hunk 3]
```diff
@@ -19,18 +19,22 @@ const BADGE_LABELS = {
...
+    const startTime = get(activity, 'object.startTime')
+    const isUpcoming = new Date(startTime) > new Date()
+    return isUpcoming ? STATUS.UPCOMING : STATUS.INCOMPLETE
   }
+
+  if (apiStatus === STATUS.COMPLETE) {
+    return STATUS.COMPLETE
+  }
+
+  return null
```
Comment (yeahfro) [inline src/activity-feed/activity-feed-cards/InteractionUtils.js]: Do we need this `return null` here? The interaction should always be `draft` or `complete` so we get to this point, returning `undefined` is okay.

Comment (rafenden) [reply src/activity-feed/activity-feed-cards/InteractionUtils.js]: We don't need to return `null`, it's a just a good practise and currently it's one of the ESLint rules

**Pull request merged**